share/completions/castnow.fish

```diff
@@ -0,0 +1,30 @@
+set -l __fish_castnow_keys "space\tToggle\ between\ play\ and\ pause m\tToggle\ mute t\tToggle\ subtitles up\tVolume\ up down\tVolume\ down left\tSeek\ backward right\tSeek\ forward n\tNext\ in\ playlist s\tStop\ playback quit\tQuit"
```

Comment:
Bingo! Your analysis is correct. I missed the single versus double-quote difference. Thank you for thinking about this and responding to feedback in a thoughtful manner. I prefer the idiom in your change. I think the other, similar, patterns should match this one for consistency. But that's for another issue/pull-request.